.github/workflows/cypress-firefox.yml

```
@@ -0,0 +1,22 @@
+name: Cypress push and pull
+
+on: [push, pull_request]
+
+jobs:
+  cypress-run:
+     name: firefox
+     runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: update test fixture
+        run: npm run pre:cy:genfix
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          start: npm run serve
+          wait-on: http://127.0.0.1:8080
+          wait-on-timeout: 1200
+          browser: firefox
```

The last thing I noticed is that we need to run Cypress in headless mode:

```suggestion
          browser: firefox
          headless: true
```